### ISSUE TITLE
Maximum data refresh rate is about 250Hz???

### ISSUE THREAD

**Issue by rtek1000** [imxieyi/esp32-i2c-mpu6050#4]
Datasheet says (page 17):

> Gyroscope Sample Rate, Fast: 8 kHz

> Accelerometer Sample Rate: 1 kHz

What if we read only what matters?

For example, if we read all the data, it will take a longer time than reading only one axis!

**Comment by imxieyi** [comment]
Of course you can read only the data you need. According to datasheet, MPU60X0's maximum SCL clock is 400khz, although ESP32 supports maximum of 2mhz. In my practice when reading all the data the data rate is at most 250hz.

**Comment by rtek1000** [comment]
Yes, but to say it in a few words may be enough for the unsuspecting to discard the use of this sensor, which is why I reported the case.
